ner_telhub/widgets/menu_widgets.py

```diff
@@ -76,5 +78,39 @@ def __init__(self, parent: QWidget):
         layout = QVBoxLayout()
         layout.addWidget(table)
         self.setLayout(layout)
-        
+
+
+class ButtonStyles(Enum):
```

Comment:
I think a better design is to make this an inner class of NERButton, and maybe to rename it to just Styles. Then we can make the init method of NERButton take in a Style instead of a string. The new call to create a button would look like `button = NERButton("Filter", NERButton.Styles.RED)`